README.md

```diff
@@ -257,6 +257,7 @@ Variables:
 - `stack` - Kernel stack trace
 - `ustack` - User stack trace
 - `arg0`, `arg1`, ... etc. - Arguments to the function being traced
+- `sarg0`, `sarg1`, ... etc. - Arguments to the function being traced (for Go programs built with `gc`)
```

Comment:
The go part is a bit confusing here, unless you know what `go gc` is. I would reword it to make it clear that it is reading from the stack.